redis/5.4/modules/ROOT/pages/redis-connector-xml-maven.adoc

```diff
@@ -0,0 +1,72 @@
+= Redis Connector 5.4 XML and Maven Support - Mule 4
+:page-aliases: connectors::redis/redis-connector-xml-maven.adoc
+
+If you manually code a Mule runtime engine (Mule) app in XML, either from the Anypoint Studio XML editor or from a text editor, you can get access to the connector in your app by adding reference statements to both your XML Mule flow and the Apache Maven `pom.xml` file.
+
+When you install the connector in Studio, Studio automatically populates the XML code with the connector's namespace and schema location. It also adds a dependency to the `pom.xml` file.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/redis
+http://www.mulesoft.org/schema/mule/redis/current/mule-redis.xsd">
+----
+
+This example shows how the namespace statements are placed in the `<mule>` XML block:
+
+[source,xml,linenums]
+----
+<mule xmlns:redis="http://www.mulesoft.org/schema/mule/redis"
+	xmlns:tracking="http://www.mulesoft.org/schema/mule/ee/tracking"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:spring="http://www.springframework.org/schema/beans"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans
+	http://www.springframework.org/schema/beans/spring-beans-current.xsd
+	http://www.mulesoft.org/schema/mule/core
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/http
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/tracking
+	http://www.mulesoft.org/schema/mule/ee/tracking/current/mule-tracking-ee.xsd
+	http://www.mulesoft.org/schema/mule/redis
+	http://www.mulesoft.org/schema/mule/redis/current/mule-redis.xsd">
+----
+
+== Add a POM File Dependency
+
+The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies for a Mule app. If you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector:
+
+[source,xml,linenums]
+----
+<dependency>
+  <groupId>com.mulesoft.connectors</groupId>
+  <artifactId>mule-redis-connector</artifactId>
+  <version>x.x.x</version>
```

Comment:
This is just a question - do we leave this blank? I have been putting at least the first number here.

Reply:
We do it exactly as shown in our template. This is correct. There is text afterwards that explains what to replace x.x.x with.